lib/get-files.js

```diff
@@ -8,11 +8,26 @@ const fileList = [
   'yarn.lock'
 ]
 
-async function getGithubFile (ghqueue, { path, owner, repo }) {
+async function getGithubFile (ghqueue, { path, owner, repo, sha }, log) {
   try {
     return await ghqueue.read(github => github.repos.getContents({ path, owner, repo }))
-  } catch (e) {
-    return { name: path, path, content: false }
+  } catch (error) {
```

Comment:
just curious, why not always use the data api, instead of only when >1MB

Reply:
because we need to make 2 requests for > 1MB files.
getTree(tree_sha) -> file_sha
getBlob(file_sha) -> content

Reply:
makes perfect sense